releasenotes/notes/submit-evaluation-for-01096d803d969e3e.yaml

```diff
@@ -0,0 +1,12 @@
+---
+features:
+  - |
+    LLM Observability: This introduces the ability to submit custom evaluations joined to a span by a tag key-value pair using the
+                      `LLMObs.submit_evaluation_for()` method. The tag key-value pair is expected to uniquely a span.
+                      Example usage: 
+                        - Evaluation joined by tag: `LLMObs.submit_evaluation_for(span_with_tag=("message_id", "dummy-message-id"), label="rating", ...)`.
+                        - Evaluation joined by trace/span ID: `LLMObs.submit_evaluation_for(span={"trace_id": "...", "span_id": "..."}, label="rating", ...)`.
+deprecations:
+  - |
+    LLM Observability: `LLMObs.submit_evaluation` is deprecated and will be removed in 3.0.0.
+                        As an alternative to `LLMObs.submit_evaluation`, you can use `LLMObs.submit_evaluation_for` instead.
```

Comment:
might be worth giving the basic example of how to migrate (just a simple example converting the call)